### PR TITLE
Updates setup.py and show version

### PR DESCRIPTION
**Pull request by holytshirt**
- Updates setup.py to be more flexible for version work
 - Show version when octopose --version #47

### CODE REVIEW

File: octopose/octopose.py [hunk 1]
```diff
@@ -23,23 +23,27 @@
...
 
-from octopose import generate_manifest, deploy
+from octopose import generate_manifest, deploy, __version__
 
 
 def main():
     parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter, prog='octopose',
                                      description='Octopose is a manifest/state driven deployment framework for Octopus Deploy.',
-                                     usage='%(prog)s [generate] [deploy] [-h]',
+                                     usage='%(prog)s [generate] [deploy]',
```
Comment (domgreen) [inline octopose/octopose.py]: would keep the `-h` or `--help` as it shows that is an option ... `git` does this

File: octopose/__version__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+#  '+,  `++        ;,                                                      
+#  +:    `+   '++  ++;  `++'    ;++`   ,++:   '++`  :++.                   
+#  +      +: ++;++.+++ `++;++  ++;++. '+''+' ++:++ '+;;+.                   
+# `+      +:.+     ;:  ++   +` +   ++ +:  ,+ ++,   +,``++                  
+#  +,     +`:+     ;:  +'   +,`+   :+ +`   +  '+++ ++++++                  
+#  ++`   ++ `+`  ` ;+  '+  `+ `+:  ++ ++  '+ ;   + +'  `                  
+#   ++++++   +++++  ++: ++++' `+++++  `++++. +++++ `++++.                 
+#     :;`     `:.    ,   .:   `+ ,,     ,,    `:.    ,,                     
+#                             `+                                                
+#                              +
```
Comment (domgreen) [inline octopose/__version__.py:12]: whats this?

File: setup.py [hunk 3]
```diff
@@ -20,36 +31,133 @@
...
+import sys
+from shutil import rmtree
+
+from setuptools import find_packages, setup, Command
+
+# Package meta-data.
+NAME = 'octopose'
+DESCRIPTION = 'Command line tools for managing local and remote octopus deploys using manifests.'
+URL = 'https://github.com/HuddleEng/octopose'
+EMAIL = 'george.ayris@huddle.com'
```
Comment (domgreen) [inline setup.py]: would update this

Comment (domgreen) [reply setup.py]: do you have `oss@huddle.com`?

File: setup.py [hunk 4]
```diff
@@ -20,36 +31,133 @@
...
+AUTHOR = 'George Ayris'
```
Comment (domgreen) [inline setup.py]: `HuddleEng`?

**Review by domgreen — COMMENTED** [review]
Would like to see this as separate PRs ... number of comments but nothing major.

File: octopose/octopose.py (hunk 1 above)
Comment (domgreen) [reply octopose/octopose.py]: add `--version`

File: octopose/__version__.py (hunk 2 above)
Comment (AndrewJanuary) [reply octopose/__version__.py:12]: ASCII art of our awesome Octopose logo.

File: setup.py (hunk 3 above)
Comment (AndrewJanuary) [reply setup.py]: I don't think we have a specific email for oss but you're right, we should probably update it to an active huddle email at least.

File: octopose/__version__.py (hunk 2 above)
Comment (holytshirt) [reply octopose/__version__.py:12]: Octopose in ascii text :)

File: setup.py (hunk 3 above)
Comment (holytshirt) [reply setup.py]: yeah I find out what it is

File: octopose/__version__.py (hunk 2 above)
Comment (holytshirt) [reply octopose/__version__.py:12]: look at the whole file :)

**Pull request merged**